graphs/templates/base.html

```diff
@@ -109,7 +109,10 @@ <h4 class="modal-title" id="myModalLabel">Forgot Information</h4>
                     {% endif %}
                     <li><a href="{% url 'features' %}">Features</a></li>
                     <li><a href="{% url 'help_tutorial' %}">Help</a></li>
-                    <li><a href="{% url 'help_about' %}" style="text-align:center">About Us</a></li>                     
+                    <li><a href="{% url 'help_about' %}" style="text-align:center">About Us</a></li>  
+                    {% if uid != None %}
```

Comment:
Yes this will be enough as `uid` gets a value only if the user is logged in and a valid user. Logic of this https://github.com/Murali-group/GraphSpace/blob/master/graphs/auth/login.py#L44